src/database-fs.ts

```diff
@@ -20,26 +20,33 @@ export async function clear (): Promise<void> {
 
 export async function count (): Promise<number> {
   let n = 0
-  for await (const _entry of Deno.readDir(dataFolder)) {
-    n++
+  for await (const entry of Deno.readDir(dataFolder)) {
+    // Explicit check to satisfy strict-boolean-expressions lint rule
+    if (entry.isFile === true) {
```

Comment:
I am not a fan of this `strict-boolean-expressions` rule. It is causing ugly modifications throughout the code.

Could you keep the check as `if (entry.isFile) {` and place `strict-boolean-expressions` on the ignore list?

Reply:
`!!entry.isFile` would be simpler but I'm not sure why this is (was) even an issue here, considering that according to the [docs](https://docs.deno.com/api/deno/~/Deno.DirEntry) this should be a boolean to begin with.